tests/scripts/test_stdlib.py

```diff
@@ -0,0 +1,7 @@
+from subprocess import check_output, CalledProcessError
+from leapp.libraries.stdlib import check_output as tested_check_output
+
+
+def test_check_output():
+    a_command = ['pwd']
+    assert check_output(a_command) == tested_check_output(a_command)
```

Comment:
That's not sufficient testing. Actually, it doesn't mean anything. You should check, you will get specific output, not the same output that provides original `check_output` function. This test doesn't tell you anything. Better would be to use even `['echo', '<string>']` and check you get '<string>'. In addition, the function has additional parameter, you don't test it. We expect that library functions will be well tested. So I expect more tests, that will show the correct functionality of the function.